manifests/config/exporter/kustomization.yaml

```diff
@@ -0,0 +1,39 @@
+resources:
+# uncomment this line for openshift
+# - openshift_scc.yaml
+- exporter.yaml
+# uncomment this line if prometheus deployed
+# - prometheus_service_monitor.yaml
+
+patchesStrategicMerge: []
+# add this line to allow ci
+# - ./patch/patch-ci.yaml
```

Comment:
please check `patch-ci.yaml` and add this to the patch so that CI can mount `/usr/src`

```
      containers:
      - name: kepler-exporter
        volumeMounts:
        - mountPath: /usr/src
          name: usr-src
      volumes:
      - name: usr-src
        hostPath:
          path: /usr/src
          type: Directory
```

Reply:
is the /usr/src is mandatory in the base image? I see it doesn't exist here https://github.com/sustainable-computing-io/kepler/blob/main/manifests/kubernetes/base/deployment.yaml, so I didn't put it in the main image.
Is this patch needed for the integration test? If so, I will try adding the option CI_DEPLOY when building the manifest.

Reply:
Adding CI_DEPLOY option really solves the issue. Thanks. 
Note that, with this PR, there is no need to separate deployment for ci too. So, the following TO-DO and NOTE is clear now.

https://github.com/sustainable-computing-io/kepler/blob/daf15534d49cfa610ec8a61514e7129265795354/cluster-up/cluster/kind/common.sh#L171-L172